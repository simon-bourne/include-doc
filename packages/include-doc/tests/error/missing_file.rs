use include_doc::source_file;

#[doc = source_file!("non_existant_file.rs")]
fn main() {}
