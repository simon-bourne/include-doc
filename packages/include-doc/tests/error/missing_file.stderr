error: No such file or directory (os error 2)
 --> tests/error/missing_file.rs:3:22
  |
3 | #[doc = source_file!("non_existant_file.rs")]
  |                      ^^^^^^^^^^^^^^^^^^^^^^
